src/styles/_catalog.scss

```diff
@@ -1,9 +1,10 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-auto-flow: row;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
```

Comment:
You can use variables for card-width, in reason that this value was using several times (in catalog and in catalog__card)